### PR TITLE
Drop the package virttest.staging.backports

### PR DESCRIPTION
**Pull request by luckyh**
Since legacy python (2.4-2.6) is no longer supported by avocado-vt,
now it is the time to drop the package `virttest.staging.backports`
as it is used for backward compatibility.

Signed-off-by: Xu Han <xuhan@redhat.com>

ID: 1607758

### CODE REVIEW

**Comment by luckyh** [comment]
Depends on:
https://github.com/autotest/tp-libvirt/pull/1793
https://github.com/autotest/tp-qemu/pull/1465

**Comment by luckyh** [comment]
The package affects the installation of py3:
```
Processing avocado_plugins_vt-63.0-py3.6.egg
removing '/usr/local/lib/python3.6/site-packages/avocado_plugins_vt-63.0-py3.6.egg' (and everything under it)
creating /usr/local/lib/python3.6/site-packages/avocado_plugins_vt-63.0-py3.6.egg
Extracting avocado_plugins_vt-63.0-py3.6.egg to /usr/local/lib/python3.6/site-packages
  File "/usr/local/lib/python3.6/site-packages/avocado_plugins_vt-63.0-py3.6.egg/virttest/staging/backports/simplejson/encoder.py", line 338
    False=False,
        ^
SyntaxError: invalid syntax
```

**Comment by luckyh** [comment]
@ldoktor @clebergnu @balamuruhans please help review this one, thanks!

**Comment by balamuruhans** [comment]
@luckyh so, still avocado-vt supports python 2.7 right ?

File: virttest/qemu_devices/qdevices.py [hunk 1]
```diff
@@ -9,21 +9,15 @@
 import logging
 import re
 import traceback
+# pylint: disable=E0611
```
Comment (clebergnu) [inline virttest/qemu_devices/qdevices.py]: I don't see the reason for this pylint exception.  On my environment:

```
$ git diff
diff --git a/virttest/qemu_devices/qdevices.py b/virttest/qemu_devices/qdevices.py
index 5b2957459..f368f67e9 100644
--- a/virttest/qemu_devices/qdevices.py
+++ b/virttest/qemu_devices/qdevices.py
@@ -9,7 +9,6 @@ interact with qemu qdev structure.
 import logging
 import re
 import traceback
-# pylint: disable=E0611
 from collections import OrderedDict
 
 from virttest import qemu_monitor

$ inspekt lint --disable W,R,C,E1002,E1101,E1103,E1120,F0401,I0011,E1003 virttest/qemu_devices/qdevices.py
Pylint disabled: W,R,C,E1002,E1101,E1103,E1120,F0401,I0011,E1003
Pylint enabled : W0611
Syntax check PASS
```

Comment (luckyh) [reply virttest/qemu_devices/qdevices.py]: ok, that makes sense.

**Pull request merged**